src/components/ReviewsList.jsx

```diff
@@ -1,14 +1,19 @@
+import { useEffect } from "react";
 import ReviewCard from "./ReviewCard";
+import { getReviews } from "../api";
+
+const ReviewsList = ({ reviews, setReviews }) => {
```

Comment:
There should be an indication for the user that your data is loading. 

Think about the isLoading pattern and try to apply it here (and anytime you're waiting for data 😄)